src/ontology/cl-edit.owl

```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033040 "lung resident memory CD8-positive, CD103-positive, alpha-beta T cell")
 EquivalentClasses(obo:CL_4033040 ObjectIntersectionOf(obo:CL_4033039 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001010)))
 
+# Class: obo:CL_4033041 (alveolar macrophage CCL3-positive)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033041 "An alveolar macrophage that expresses CCL3.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033041 "2023-06-15T12:53:08Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033041 "alveolar macrophage CCL3-positive")
+SubClassOf(obo:CL_4033041 obo:CL_0000583)
+
+# Class: obo:CL_4033042 (alveolar macrophage metallothionein-positive)
```

Comment:
Labels in CL tend to have markers at the start of the label, so consider relabelling to 'metallothionein-positive alveolar macrophage'.